Sources/EpoxyBars/BarInstaller/SelectableStackView.swift

```diff
@@ -0,0 +1,102 @@
+// Created by matthew_cheok on 2/4/21.
+// Copyright © 2021 Airbnb Inc. All rights reserved.
+
+import EpoxyCore
+import UIKit
+
+/// A stack of arbitrary bar views that can handle user selection.
+public final class SelectableStackView: BarStackView, StyledView {
```

Comment:
We should just be able to have this conform to `EpoxyableView` instead of `StyledView` and `ContentConfigurableView`
```suggestion
public final class SelectableStackView: BarStackView, EpoxyableView {
```